src/main.ts

```diff
@@ -292,6 +302,8 @@ export class Purchases {
         rcPackage: pkg,
         htmlTarget: paywallParams.purchaseHtmlTarget,
         customerEmail: paywallParams.customerEmail,
+        selectedLocale: selectedLocale,
+        defaultLocale: offering.paywall_components?.default_locale || "en",
```

Comment:
Nitpick but might be worth extracting `en` to a constant? Like `const DEFAULT_LOCALE = "en"`. I see we also use this in a few places so probably reuse it in those.

Reply:
Done, called it `englishLocale` just to avoid colliding with `defaultLocale`.

It's not a best practice to call constant variables with all caps in JS.